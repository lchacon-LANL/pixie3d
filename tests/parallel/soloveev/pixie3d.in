 &setup
    nxd      = 32
    nyd      = 32
    nzd      = 32

    xmax     = 1d0
    ymax     = 0.
    zmax     = 0.

    numerical_grid = t

    bcs      = 'spt','def','per','per','per','per'

    coords   = 'ext'
 /

 &phys
    eta      = 1d-3
    nu       = 1d-2
    dd       = 0d-3
    chi      = 0d-3

    dlambda  = 0.5

    gamma    = 1.6667

    adiabatic  = t

    pvx      = 1d-4
    odd      = f
    random   = f

    nh2      = 1
    nh3      =-2

    equil    = 'vmec'
 /

 &fv
    sym_st   = f

    advect   = 4
    v_advect = 4

    solenoidal = t
    nc_eom_f   = t
    nc_eom_v   = f

    solve_rho  = f
    use_p_eom  = t
    use_p_diff = t
 /

 &ts
    dt       = 5d-4

    tmax     = 5d-3
    dstep    = 5d-3

    ts_restart  = f
    ts_adapt_dt = f

    cnfactor = 0d0
    sm_flag  = 2

    source   = t

 /

 &nks
    maxksp   = 20
    maxitgm  = 20
    tolgm    = 8d-1
    rtol     = 1d-4
    atol     = 0d0
    method   = 1
    mf_eps   = 1d-6
    vol_wgt  = t
    iguess   = 0
 /

 &pc
    pc_type   = 'id'
    pc_tol    = 1d-1
    pc_iter   = 0
    mg_vcyc   = 2
    sm_iter   = 4
 /

 &i_o
    ilevel   = 0
    dcon     = f
    check_grid = f
    chk_src  = f
    test     = t
    equ_file = 'wout_soloveev_k=1.txt'
 /

 &graphics
    sel_diag  = 2,12,15,17,18,19,20,22,23
    dplot = 1d-4
    prof_conf = 1,'r',1,0,1
    cont_conf = 3,'R','Z',1,1,1
!    cont_conf = 2,'R','PHI',1,1,1
    hdf_plot  = t
    xdraw_plot= t
/

&plots
    poincare  = f
    clean = f
 /

