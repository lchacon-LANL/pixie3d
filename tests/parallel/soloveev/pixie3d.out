 Reading VMEC map on grid  1, nx x ny x nz= 32x 32x 32
 Reading VMEC map on grid  2, nx x ny x nz= 16x 16x 16
 Reading VMEC map on grid  3, nx x ny x nz=  8x  8x  8
 Reading VMEC map on grid  4, nx x ny x nz=  4x  4x  4
  >>> Coarsening VMEC mesh hierarchy
    >>> New coarse grid hierarchy

 Reading VMEC solution...

  Grid mesh:................   32x   32x   32
  Number of processors:.....    2,    2,    2
  Preconditioning method:... Identity
  Time correction...........   F
  External source...........   T
  Numerical grid............   T
  Scalar advection scheme...  SMART   
  Velocity advection scheme.  SMART   
  TS scheme................. BDF2     

    itime      time     dt     #CFL   vx_max   vy_max   vz_max    GMRES Newton CN factor  GMRES-DC
         1      0.00  5.0E-05 1.5E+00  1.2E-04  7.2E-08  7.5E-10     3     3     0.000        0
         2      0.00  4.5E-04 1.3E+01  1.2E-04  2.5E-06  2.7E-08    11     5     0.000        0
         3      0.00  5.0E-04 1.5E+01  1.2E-04  8.0E-06  9.9E-08    17     6     0.000        0
         4      0.00  5.0E-04 1.5E+01  1.2E-04  1.5E-05  2.1E-07    19     6     0.000        0
         5      0.00  5.0E-04 1.5E+01  1.1E-04  2.3E-05  3.7E-07    18     6     0.000        0
         6      0.00  5.0E-04 1.5E+01  1.1E-04  3.2E-05  5.6E-07    18     6     0.000        0
         7      0.00  5.0E-04 1.5E+01  1.0E-04  4.3E-05  7.8E-07    19     6     0.000        0
         8      0.00  5.0E-04 1.5E+01  9.7E-05  5.5E-05  1.1E-06    18     6     0.000        0
         9      0.00  5.0E-04 1.5E+01  9.0E-05  7.0E-05  1.5E-06    18     6     0.000        0
        10      0.00  5.0E-04 1.5E+01  8.3E-05  8.6E-05  2.0E-06    18     6     0.000        0
        11      0.01  5.0E-04 1.5E+01  7.5E-05  1.0E-04  2.5E-06    18     6     0.000        0

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   11       5.6       16.1        2.9
